reversinglabs_a1000v2.json

```diff
@@ -0,0 +1,1574 @@
+    "utctime_updated": "2023-04-05T05:26:10.159217Z",
+    "package_name": "phantom_reversinglabs_a1000v2",
+    "main_module": "reversinglabs_a1000v2_connector.py",
+    "min_phantom_version": "5.5.0",
+    "fips_compliant": false,
+    "app_wizard_version": "1.0.0",
+    "configuration": {
+        "url": {
+            "description": "A1000 url",
+            "data_type": "string",
+            "required": true,
+            "order": 0,
+            "name": "url",
+            "id": 0
+        },
+        "token": {
+            "description": "A1000 token",
+            "data_type": "password",
+            "required": true,
+            "order": 1,
+            "name": "token",
+            "id": 1
+        }
+    },
+    "actions": [
+        {
+            "action": "test connectivity",
+            "identifier": "test_connectivity",
+            "description": "Validate the asset configuration for connectivity using supplied configuration",
+            "verbose": "Validate the asset configuration for connectivity using supplied configuration.",
+            "type": "test",
+            "read_only": true,
+            "parameters": {},
+            "output": [],
+            "versions": "EQ(*)"
+        },
+        {
+            "action": "detonate file",
+            "identifier": "detonate_file",
+            "description": "Upload file to A1000",
+            "verbose": "Upload file to A1000.",
+            "type": "investigate",
+            "read_only": true,
+            "parameters": {
+                "vault_id": {
+                    "description": "Vault ID of file to detonate",
+                    "data_type": "string",
+                    "required": true,
+                    "primary": true,
+                    "contains": [
+                        "pe file",
+                        "pdf",
+                        "flash",
+                        "apk",
+                        "jar",
+                        "doc",
+                        "xls",
+                        "ppt"
+                    ],
+                    "order": 0,
+                    "name": "vault_id"
+                },
+                "file_name": {
+                    "description": "Filename to use",
+                    "data_type": "string",
+                    "required": false,
+                    "primary": false,
+                    "contains": [
+                        "file name"
+                    ],
+                    "order": 1,
+                    "name": "file_name"
+                }
+            },
+            "output": [
+                {
+                    "data_path": "action_result.status",
+                    "data_type": "string",
+                    "column_name": "status",
+                    "column_order": 2
+                },
+                {
+                    "data_path": "action_result.parameter.vault_id",
+                    "data_type": "string",
+                    "contains": [
```

Comment:
Can these be in alphabetical order?

Reply:
Organized the data paths in alphabetical order. Please review it.